packages/json-extension/src/component.tsx

```diff
@@ -9,7 +9,7 @@ import { JSONArray, JSONExt, JSONObject, JSONValue } from '@lumino/coreutils';
 import * as React from 'react';
 import Highlight from 'react-highlighter';
 import { JSONTree } from 'react-json-tree';
-
+import { StyleModule } from 'style-mod';
```

Comment:
Looks like the `style-mod` dependency should be added to the `package.json`, as suggested by the following check failure:

https://github.com/jupyterlab/jupyterlab/actions/runs/3525395365/jobs/5912584760

Reply:
Ah my bad your right. Will add.